kie-server-parent/kie-server-tests/kie-server-integ-tests-dmn/pom.xml

```diff
@@ -0,0 +1,267 @@
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.server</groupId>
+      <artifactId>kie-server-rest-dmn</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
+    <!-- Logging -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <!-- REST --> 
+    <dependency>
+      <groupId>org.jboss.spec.javax.servlet</groupId>
+      <artifactId>jboss-servlet-api_3.1_spec</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>tjws</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+   <!--  Stay with httpcomponents: it will save us pain (as opposed to changing from resteasy to cxf to cxf 3.0, etc. --> 
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>commons-logging</groupId>
+          <artifactId>commons-logging</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>fluent-hc</artifactId>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>commons-logging</groupId>
+          <artifactId>commons-logging</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <!-- Replacement for the above excluded 'commons-logging:commons-logging' -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jcl-over-slf4j</artifactId>
+      <scope>test</scope>
+    </dependency>
+   
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-embedder</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.kie.server</groupId>
+      <artifactId>kie-server</artifactId>
+      <classifier>webc</classifier>
+      <type>war</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.server</groupId>
+      <artifactId>kie-server</artifactId>
+      <classifier>ee6</classifier>
+      <type>war</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.server</groupId>
+      <artifactId>kie-server</artifactId>
+      <classifier>ee7</classifier>
+      <type>war</type>
+      <scope>test</scope>
+    </dependency>
+    
+    <!-- DO NOT REMOVE THIS COMMENT. Ref commit 3a92c62 of PR droolsjbpm/droolsjbpm-integration#447 
```

Comment:
@etirelli this is finally green passing Jenkins CI; I details my finding for posterity and hopefully shall help others or our future selves for reference.

The error reported:
```
23:46:06 [INFO] KIE :: Execution Server :: Tests ................... SUCCESS [  1.731 s]
23:46:06 [INFO] KIE :: Execution Server :: Tests :: Common Integration Tests SUCCESS [02:48 min]
23:46:06 [INFO] KIE :: Execution Server :: Tests :: Drools Integration Tests SUCCESS [02:59 min]
23:46:06 [INFO] KIE :: Execution Server :: Tests :: jBPM Integration Tests SUCCESS [12:11 min]
23:46:06 [INFO] KIE :: Execution Server :: Tests :: OptaPlanner Integration Tests FAILURE [02:15 min]
23:46:06 [INFO] KIE :: Execution Server :: Tests :: Drools and jBPM Integration Tests SKIPPED
23:46:06 [INFO] KIE :: Execution Server :: Tests :: Controller Integration Tests SKIPPED
23:46:06 [INFO] KIE :: Execution Server :: Tests :: Policies Integration Tests SKIPPED
23:46:06 [INFO] KIE :: Execution Server :: Tests :: Router Integration Tests SKIPPED
23:46:06 [INFO] KIE :: Execution Server :: Tests :: Case Id Generator Integration Tests SKIPPED
23:46:06 [INFO] KIE :: Execution Server :: Tests :: DMN Integration Tests SUCCESS [01:02 min]
23:46:06 [INFO] KIE :: OSGi Integration Parent Project ............. SUCCESS [  0.477 s]
23:46:06 [INFO] kie-karaf-itests-domain-model ...................... SUCCESS [  8.390 s]
23:46:06 [INFO] KIE :: Karaf Integration Tests Kjar ................ SUCCESS [  8.719 s]
23:46:06 [INFO] KIE :: OSGi Integration ............................ SUCCESS [ 10.653 s]
23:46:06 [INFO] KIE :: Karaf Features .............................. SUCCESS [  9.899 s]
23:46:06 [INFO] KIE :: Karaf Integration Tests ..................... SUCCESS [02:54 min]
23:46:06 [INFO] KIE Performance Kit ................................ SUCCESS [ 17.782 s]
23:46:06 [INFO] Drools :: Android Integration ...................... SUCCESS [ 17.497 s]
23:46:06 [INFO] kie-security ....................................... SUCCESS [  8.513 s]
23:46:06 [INFO] Drools and jBPM integration distribution ........... SUCCESS [  7.143 s]
23:46:06 [INFO] BPMS distribution .................................. SUCCESS [ 17.775 s]
23:46:06 [INFO] BRMS distribution .................................. SUCCESS [  9.288 s]
23:46:06 [INFO] ------------------------------------------------------------------------
23:46:06 [INFO] BUILD FAILURE
23:46:06 [INFO] ------------------------------------------------------------------------
23:46:06 [INFO] Total time: 27:52 min (Wall Clock)
23:46:06 [INFO] Finished at: 2017-03-02T18:46:05-05:00
23:46:07 [INFO] Final Memory: 510M/1609M
23:46:07 [INFO] ------------------------------------------------------------------------
23:46:07 [ERROR] Failed to execute goal org.codehaus.cargo:cargo-maven2-plugin:1.6.2:stop (stop-container) on project kie-server-integ-tests-optaplanner: Execution stop-container of goal org.codehaus.cargo:cargo-maven2-plugin:1.6.2:stop failed: Failed to stop the WildFly 10.0.0.Final container. Failure when invoking CLI script, java returned 1 -> [Help 1]
```
had nothing to to with `kie-server-integ-tests-optaplanner` because with this PR I didn't change anything about optaplanner modules.

I then realised the real problem was that `integ-test-xxx` reactor modules need to be executed sequentially, otherwise they would fail for conflicting ports. The way to achieve this is to have each `integ-test-xxx` reactor modules with an "artificial dependency", which is implemented with the lines here.
Ref commit 3a92c62 of PR droolsjbpm/droolsjbpm-integration#447 

In this case, I looked up the actual reactor module sequence, because this `integ-test-dmn` comes after  `kie-server-integ-tests-case-id-generator` in the reactor sequence, the artificial dependency in this dmn module reference case-id-generator, as line 178.

What took me time to figure it out, is that another commit 22f0dc0c may have accidentally removed the comment explaining this "trick" from the one existing pom which I used as a reference during my development, so I'll raise separately to instantiate the comment back (and the reason in this pom I've put the explicit reference possibly helpful for posterity).